src/doom/p_mobj.c

```diff
@@ -38,7 +38,7 @@
 
 
 void G_PlayerReborn (int player);
-void P_SpawnMapThing (mapthing_t*	mthing);
+void P_SpawnMapThing (mapthing_t*	mthing, int index);
```

Comment:
Conflicting definitions are not allowed. Function is defined with second parameter, so all forward declares must have it as well.

Reply:
Looks like in windows I didnt get error/warning for that. weird! Thanks